sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/README.md

```diff
@@ -0,0 +1,48 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-schema-registry-avro-serializer
+urlFragment: schemaregistry-avro-serializer-samples
+---
+
+# Azure Schema Registry Avro Serializer library for Python Samples
+
+These are code samples that show common scenario operations with the Schema Registry Avro Serializer library.
+The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations, 
+and require Python 3.5 or later.
+
+Several Schema Registry Avro Serializer Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Schema Registry Avro Serializer:
+
+* [avro_serializer.py](./sync_samples/avro_serializer.py) - Examples for common Schema Registry Avro Serializer tasks:
+    * Serialize data according to the given schema
+    * Deserialize data
+
+## Prerequisites
+- Python 2.7, 3.5 or later.
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Schema Registry, you'll need a subscription.
+If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+## Setup
+
+1. Install the Azure Schema Registry client library for Python with [pip](https://pypi.org/project/pip/):
```

Comment:
It's not actually "Azure Schema Registry client library" that they install

Reply:
too many flaws in my doc ><, thanks!